### PR TITLE
Add Percentage in Enrolled Column

### LINKED ISSUE #56: Add percentage for enrolled column

### PR DESCRIPTION
**Pull request by masonma21**
In this PR, we add the fraction for the enrolled column. 
We write a new function that takes the numerator (enrolled) and denominator (maximum) and return the percentage to the enrolled column. 
In this case, the enrollment column will display in a fraction of the total enrolled over the max seats available and give a specific percentage. Some rows will return empty since the enrolled seats or the maximum seats are invalid. It passed all the tests.

Here is the deployment on QA:

image

Close #56

### CODE REVIEW

**Review by Abelatnafu — DISMISSED** [review]
LGTM

**Comment by taeseongk** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,6 @@ frontend/reports
 frontend/strykerTmp
 /.stryker-tmp
 
-
```
Comment (andrewpengucsb) [inline .gitignore]: Even though this is just removal of a new line, you dont want to be merging changes to any file that is not supposed to change.

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (masonma21) [reply .gitignore]: Pulled the .gitignore from the main and commit it.

**Review by Abelatnafu — APPROVED** [review]
LGTM!

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**